GameData/RealismOverhaul/Engine_Configs/AJ10_190_Config.cfg

```diff
@@ -30,9 +30,9 @@
 
     @MODULE[ModuleGimbal],*
     {
-        @gimbalRange = 6.0
-        %useGimbalResponseSpeed = True
-        %gimbalResponseSpeed = 16
+        gimbalRange = 6.0
```

Comment:
did you try using the `%` modifier? that will edit or create, depending on if it already exists.